src/neo/Ledger/HeaderHashList.cs

```diff
@@ -1,35 +0,0 @@
-using Neo.IO;
-using System.IO;
-
```

Comment:
Can we split this PR into two PRs？ One is to remove unused code, and another is to move everything into LedgerContract?

Reply:
This file becomes unused because of the moving.